[Excel] (Workbook) Update insert worksheets article section with new Preview API (#2554)
OfficeDev/office-js-docs-pr: docs/excel/excel-add-ins-workbooks.md

### Insert a copy of an existing workbook into the current one (preview)

> [!NOTE]
> The `Workbook.insertWorksheetsFromBase64` method is currently only available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
> 

The previous example shows a new workbook being created from an existing workbook. You can also copy some or all of an existing workbook into the one currently associated with your add-in. A [Workbook](/javascript/api/excel/excel.workbook) has the `insertWorksheetsFromBase64` method to insert copies of the target workbook's worksheets into itself. The other workbook's file is passed as a base64-encoded string, just like the `Excel.createWorkbook` call. 

```TypeScript
insertWorksheetsFromBase64(base64File: string, options?: Excel.InsertWorksheetOptions): OfficeExtension.ClientResult<string[]>;
```

> The following example shows an existing workbook's worksheets being inserted in the current workbook, directly after the active worksheet. Note that `[]` is passed as the parameter for the [InsertWorksheetOptions](/javascript/api/excel/excel.insertworksheetoptions) `sheetNamesToInsert` property. This means that all the worksheets from the existing workbook should be inserted into the current workbook.

Reviewer: Suggestion to remove the possessive (which helps localization). Also changing the "should" to something more definitive.
```suggestion
The following example inserts another workbook in the current workbook. The new worksheets are inserted after the active worksheet. Note that `[]` is passed as the parameter for the [InsertWorksheetOptions](/javascript/api/excel/excel.insertworksheetoptions) `sheetNamesToInsert` property. This means that all the worksheets from the existing workbook are inserted into the current workbook.
```

Author: Great suggestions, thanks Alex!

> [!IMPORTANT]

> > The `insertWorksheetsFromBase64` method is supported for Excel on Windows, Mac, and the web. It's not supported for iOS. In addition, in Excel on the web this method doesn't support source worksheets with PivotTable, Chart, Comment, or Slicer elements. Using the `insertWorksheetsFromBase64` method to insert worksheets with these unsupported features in Excel on the web will return the `UnsupportedFeature` error. 

Reviewer: Suggestions for brevity and removing the "-ing" verb (as per the style guide).
```suggestion
> The `insertWorksheetsFromBase64` method is supported for Excel on Windows, Mac, and the web. It's not supported for iOS. Additionally, in Excel on the web this method doesn't support source worksheets with PivotTable, Chart, Comment, or Slicer elements. If those objects are present, the `insertWorksheetsFromBase64` method Excel on the web returns the `UnsupportedFeature` error. 
```
